### ISSUE TITLE
Suggestion Manage files - Filter Category / Manage Status

### ISSUE THREAD

**Issue by sqrobert** [ignacionelson/ProjectSend#345]
First of all, really appreciate your project, nicely done and professional look!

Working with it I hit 2 actual needs that really loosen a lot of work from someone's shoulders if they're actually working with a high number of files or clients. I've seen these issues through the posts raised from other users.

1. When managing as an admin from "Manage files" it would be very useful to have a category filter so we can sort all the files. Users don't have a default category like "Uncategorized" so they will most probably omit to specify a category themselves. 
2. When managing as an admin from "Manage files"  it would be very useful to also have a Show/Hide like it was specified here [#66](https://github.com/ignacionelson/ProjectSend/issues/66). Also a status tab to check if it actually can be seen by the user. 

Hope this helps everyone!

**Comment by rbandak** [comment]
Why this issue did not solved yet? I cannot change hidden status to show.

**Comment by jeffkshaw** [comment]
I agree.  I need to be able to upload a file without generating notifications, but for the file to be visible to the assigned clients and client groups.

**Comment by jeffkshaw** [comment]
I was able to resolve this by connecting to the SQL server, manually editing the XXXXX**files_relations** table and **changing the "hidden" value from 1 to 0**.  Would be nice to be able to do this for each file via the web.

**Comment by ignacionelson** [comment]
This issue is being closed as part of a bulk cleanup. ProjectSend has undergone significant changes since this was reported (including r1720 and r1994), and many older issues no longer apply to the current codebase.

If you are still experiencing this problem on the latest version, please open a new issue with updated details and we'll be happy to look into it. Thank you for your contribution!
